### PR TITLE
D9 drevops branch

### PR DESCRIPTION
**Pull request by richardgaunt**
## Background

A little start on D9 drevops. The build is working successfully, tests are running successfully (behat tests are failing but due to D8  demo database I think).

## What has changed
1. Updated lagoon images to latest
2. Updated php version to 8.0
3. Updated base tests for D9 requirements
4. Changed github organisation name to richardgaunt for demo purposes.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -233,12 +233,12 @@ DREVOPS_DEBUG=
 
 # Starting containers will suppress STDOUT output, but will still show any
 # STDERR output. Set this to 1 to allow STDOUT output.
-VERBOSE_BUILD=
+BUILD_VERBOSE=
```
Comment (richardgaunt) [inline .env:236]: Little fix of the variable name

File: .env [hunk 2]
```diff
@@ -233,12 +233,12 @@ DREVOPS_DEBUG=
...
 
 # Allows to pin DrevOps to a specific commit when updating with `ahoy update`.
 # If this is not set, the latest release of DrevOps for specified DRUPAL_VERSION
 # will be used.
-DREVOPS_COMMIT=
+DREVOPS_COMMIT=9.x-0.1.0
```
Comment (richardgaunt) [inline .env:241]: To be removed - this is to get the build working succesfully

File: .env [hunk 3]
```diff
@@ -266,3 +266,8 @@ CURL_DB_URL=https://raw.githubusercontent.com/wiki/drevops/drevops/db_d8.dist.sq
 ##### DATABASE_IMAGE=drevops/drevops-mariadb-drupal-data-demo-8.x
 #;> DATABASE_IMAGE
 #;> DEMO
+
+# Development settings
```
Comment (richardgaunt) [inline .env:270]: Settings to remove - for local build (and the minimal check is for ubuntu)

File: README.md [hunk 4]
```diff
@@ -20,7 +20,7 @@
 ### Installation
 1. Run installer:
    ```
-   curl -L https://raw.githubusercontent.com/drevops/drevops/8.x/install.php > /tmp/install.php && php /tmp/install.php --interactive; rm /tmp/install.php
+   curl -L https://raw.githubusercontent.com/richardgaunt/drevops/9.x/install.php > /tmp/install.php && php /tmp/install.php --interactive; rm /tmp/install.php
```
Comment (richardgaunt) [inline README.md:23]: Updated organisation to get build working correctly.

**Comment by AlexSkrypnyk** [comment]
suppressed by https://github.com/drevops/drevops/pull/651
commit credits preserved

**Pull request closed without merging**